txbridge/src/test/java/org/jboss/jbossts/txbridge/tests/outbound/junit/DisabledContextPropagationTests.java

```diff
@@ -209,13 +210,16 @@ public void testNoTransactionWithEnabledJTAOverWSATFeature(@ArquillianResource U
     public void testNonTransactionlServiceWithEnabledJTAOverWSATFeature(@ArquillianResource URL baseURL) throws Exception {
         List<NameValuePair> parameters = new ArrayList<NameValuePair>();
         parameters.add(new BasicNameValuePair("isCommit", "true"));
-        parameters.add(new BasicNameValuePair("isTransaction", "ture"));
+        parameters.add(new BasicNameValuePair("isTransaction", "true"));
         parameters.add(new BasicNameValuePair("isWSATService", "false"));
         parameters.add(new BasicNameValuePair("isJTAOverWSATFeatureEnabled", "true"));
         parameters.add(new BasicNameValuePair("clientType", String.valueOf(TestATClient.CLIENT_WITH_JTA_FEATURE)));
-
-        List<String> invocations = makeRequest(baseURL, parameters);
-        assertInvocations(invocations);
```

Comment:
Test wasn't fail before because there was a typo in the test configuration:
parameters.add(new BasicNameValuePair("isTransaction", "ture")); // ture instead of true
After fixing this typo, an exception is thrown from the line:
makeRequest(baseURL, parameters); 
But the test pass thanks to old school try-catch. If you want I can replace try-catch to newer @Test(expected = ...)